backend/src/impl/benchmark_configs/summverse.json

```diff
@@ -0,0 +1,436 @@
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "multi-doc-summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "amasum",
+          "sub_dataset": "pros-ref",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "multi-doc-summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "amasum",
+          "sub_dataset": "cons-ref",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "multi-doc-summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "ami",
+          "sub_dataset": "document",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "arxiv_sum",
+          "sub_dataset": "section",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "arxiv_sum",
+          "sub_dataset": "document",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "citesum",
+          "sub_dataset": "document",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "cnn_dailymail",
+          "sub_dataset": "3.0.0",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "cnn_dailymail",
+          "sub_dataset": "1.0.0",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
+          ],
+          "task": "summarization",
+          "output_file_type": "text"
+      },
+      {
+          "dataset_name": "cnn_dailymail",
+          "sub_dataset": "2.0.0",
+          "dataset_split": "test",
+          "metrics": [
+              {
+                  "name": "rouge2"
+              }
```

Comment:
Is it intentional to have all three versions of CNN-DM here? I think 1.0.0 isn't even really a summarization dataset?

Reply:
Yes, we aim to include all sub-datasets, which probably differ in pre-processings, task formulations, and languages.

>"I think 1.0.0 isn't even really a summarization dataset?"

Just checked, it is a dataset.